README.md

```diff
@@ -1,6 +1,27 @@
-# langchainqna
+# Discovering inconsistencies: A deep dive with OpenAI models
+
+Industry evolves fast. Clients discover new opportunities every week.
+You are a partner in achieving their goals.
+Requirements and change-requests are coming in thick and fast.
+
+With every adaptation, you wonder if it could contradict something you've already done.
+With growth in clientele, there is no single person who understands all workflows.
+Elegant reports with five columns are slowly turning into 20-column jungles.
+
+Sounds familiar? Numocity is a player in such a field.
+
+Numocity makes SaaS products for Electric Mobility providers.
+Requirements evolve constantly, with each client in their own journey to find their business.
+
+In the quest for a single product code-base, Numocity needs to prevent the requirements from turning into a multi-header monster.
+
+This project aims to offer warnings, when product-management authors requirements that may bring inconsistency.
+
```

Comment:
detecting subtle inconsistencies and pre-evaluating scenarios of failure is a task that can be automated with the right tools, and in this modern day and age can be a weapon in your arsenal